lib/pa_ess/utilities.ex

```diff
@@ -782,18 +740,104 @@ defmodule PaEss.Utilities do
+  def audio_take({:route, "50"}), do: "686"
+  def audio_take({:route, "51"}), do: "687"
+  def audio_take({:route, "66"}), do: "585"
+  def audio_take({:route, "69"}), do: "590"
+  def audio_take({:route, "71"}), do: "591"
+  def audio_take({:route, "72"}), do: "592"
+  def audio_take({:route, "73"}), do: "594"
+  def audio_take({:route, "74"}), do: "595"
+  def audio_take({:route, "75"}), do: "596"
+  def audio_take({:route, "77"}), do: "597"
+  def audio_take({:route, "77A"}), do: "598"
+  def audio_take({:route, "78"}), do: "599"
+  def audio_take({:route, "80"}), do: "600"
+  def audio_take({:route, "86"}), do: "601"
+  def audio_take({:route, "87"}), do: "602"
+  def audio_take({:route, "88"}), do: "603"
+  def audio_take({:route, "89"}), do: "688"
+  def audio_take({:route, "90"}), do: "689"
+  def audio_take({:route, "94"}), do: "690"
+  def audio_take({:route, "96"}), do: "604"
+  def audio_take({:route, "109"}), do: "890"
+  def audio_take({:route, "170"}), do: "588"
+  def audio_take({:route, "171"}), do: "589"
+  def audio_take({:route, "226"}), do: "809"
+  def audio_take({:route, "230"}), do: "810"
+  def audio_take({:route, "236"}), do: "811"
+  def audio_take({:route, "245"}), do: "628"
+  def audio_take({:route, "716"}), do: "888"
+
+  def audio_take({:boarding, "Green-B", "70197", :boston_college}), do: "813"
+  def audio_take({:boarding, "Green-B", "70197", :kenmore}), do: "820"
+  def audio_take({:boarding, "Green-C", "70196", :cleveland_circle}), do: "814"
+  def audio_take({:boarding, "Green-C", "70196", :kenmore}), do: "823"
+  def audio_take({:boarding, "Green-D", "70199", :reservoir}), do: "815"
+  def audio_take({:boarding, "Green-D", "70199", :riverside}), do: "818"
+  def audio_take({:boarding, "Green-D", "70199", :kenmore}), do: "822"
+  def audio_take({:boarding, "Green-E", "70198", :heath_street}), do: "816"
+  def audio_take({:boarding, "Green-E", "70198", :kenmore}), do: "821"
+
+  def audio_take({:crowding, {:front, _}}), do: "870"
+  def audio_take({:crowding, {:back, _}}), do: "871"
+  def audio_take({:crowding, {:middle, _}}), do: "872"
+  def audio_take({:crowding, {:front_and_back, _}}), do: "873"
+  def audio_take({:crowding, {:train_level, :crowded}}), do: "876"
+  def audio_take({:crowding, _}), do: "21000"
+
+  def audio_take({:line, "Red"}), do: "3005"
+  def audio_take({:line, "Orange"}), do: "3006"
+  def audio_take({:line, "Blue"}), do: "3007"
+  def audio_take({:line, "Green"}), do: "3008"
+  def audio_take({:line, "Mattapan"}), do: "3009"
+  def audio_take({:line, _}), do: audio_take(:train)
+
+  def audio_take({:passthrough, :alewife, _}), do: "32114"
+  def audio_take({:passthrough, :southbound, "Red"}), do: "891"
+  def audio_take({:passthrough, :bowdoin, _}), do: "32111"
+  def audio_take({:passthrough, :wonderland, _}), do: "32110"
+  def audio_take({:passthrough, :forest_hills, _}), do: "32113"
+  def audio_take({:passthrough, :oak_grove, _}), do: "32112"
```

Comment:
Can we name the ignored parts of the pattern so it's clear what it is? This looks like a `_route_id` for example. (the `:crowding` ones are especially unclear to me)